src/Host.cpp

```diff
@@ -1390,6 +1395,14 @@ void Host::incomingStreamProcessor(const QString& data, int line)
     mTimerUnit.doCleanup();
 }
 
+// When Mudlet is running in online mode, deleted timers are cleaned up in bulk
+// on every new line. When in offline mode, new lines don't come - so they are
+// cleaned up in bulk periodically.
+void Host::slot_PurgeTimers()
```

Comment:
Should `slot_PurgeTimers()` actually be:  `slot_purgeTimers()`?

Reply:
Yeah.